scripts/fe-entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Note that the STARROCKS_HOME variable should also be set to
+# determine where to save metadata, logs, etc.
+
+mkdir -p $STARROCKS_HOME
+
+# NOTE: All FE nodes in a cluster must have the same http port:
+# https://docs.starrocks.io/docs/deployment/deploy_manually/
+
+export HTTP_PORT=${HTTP_PORT:-8030}
+export THRIFT_RPC_PORT=${THRIFT_RPC_PORT:-9020}
+export SQL_QUERY_PORT=${SQL_QUERY_PORT:-9030}
+export EDIT_LOG_PORT=${EDIT_LOG_PORT:-9010}
+
+echo "STARROCKS_HOME=${STARROCKS_HOME}"
```

Comment:
echo for debugging?

Reply:
That and makes it each to check logs if you forget ports

Reply:
👍